tests/test_sklearn_decision_tree_converter.py

```diff
@@ -444,6 +484,224 @@ def test_decision_tree_ts_classifier_converter(self):
     def test_extra_trees_ts_classifier_converter(self):
         self._run_tree_classification_converter(ExtraTreesClassifier, 3, "torch.jit", n_estimators=10)
 
+    # Test trees with TVM backend
+    # Random forest gemm classifier
```

Comment:
These test files have gotten rather long...  I'm not sure how many more we can hold! (ex: `skl->pyt, skl->ts, skl->tvm` all in the same file).  What file configuration do you think would be most straightforward for folks that might consider contributing to the project?  I know that trees are the longest. Maybe we are OK for now, but we should start planning for a refactor perhaps.

Reply:
I totally agree on this. Maybe we can split this file into decision_tree and random_forest? I think that going forward it would be good to have tests for all backends for each supported op.

Reply:
Ah, splitting to "decsion_tree"/"random_forest" seems like an easy middle ground for now!  But  it's up to you...you could leave it for now and  we could do a more expansive test refactoring later

Reply:
Added issue #358.